hedyorg/hedy
# 💻 Adding a button for Microbit

It's just there doesn't do anything at the moment.

Please replace all _____ before you create a pull request!

Start the name of the PR with one of the relevant prefixes in the title of the PR:

* 🖊️ -> changes related to grammars or the transpiler
* 🪲 -> solving a bug
* 🚚 -> changes unrelated to shipping, such as mergify scripts
* 🧹 -> refactorings (changes that do not change functionality and are only a cleanup of code)
* 💻 -> improvements and changes of the Hedy website
* 🧪 -> changes made to the test suite
* 📜 -> changes related to content

Adding a button that when pressed should transpile the Hedy code to Microbit code and then save it as a .hex file so it can be directly uploaded to Microbit. 

**How to test**

Follow these steps to verify this PR works as intended:

To test just press the button "Run Microbit" and a file should start to download. To verify it is working it can be directly copied to a Microbit device or the file can be uploaded to https://python.microbit.org/v/3

**Checklist**
Done? Check if you have it all in place using this list: (mark with x if done)


If you're unsure about any of these, don't hesitate to ask. We're here to help!
